### PR TITLE
[Storage] Update Get Last Sync Time call since rest change

### PR DESCRIPTION
**Pull request by blueww**
## Description

Update the way to get storage account Last Sync Time call since rest change. 
1. Get LST by `Get-AzureRmStorageAccount` with a new optional parameter `-IncludeGeoReplicationStats`.
2. Remove the cmdlet `Get-AzureRmStorageAccountLastSyncTime` , since the related rest API removed. (The cmdlet PR https://github.com/Azure/azure-powershell/pull/7501 is merged before, but never released.)

The PSH design doc is in: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/206

The target public preview release data is 11/15, please help to review and merge the PR.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by blueww** [comment]
@markcowl ， @maddieclayton 
Any comments for this PR?

The netcore build failed with the following error, seems not related with my code change, would you please help to see it?

> System.Management.Automation.MethodException : Cannot find an overload for "GetPropertiesWithHttpMessagesAsync" and the argument count: "4".

File: src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1 [hunk 1]
```diff
@@ -910,11 +910,6 @@ function Test-FailoverAzureStorageAccount
         Assert-AreEqual $kind $sto.Kind; 
 		$seconcaryLocation = $sto.SecondaryLocation
 
-		# check Last Sync Time
-		$getLSTResult = Get-AzureRmStorageAccountLastSyncTime -ResourceGroupName $rgname -Name $stoname
```
Comment (maddieclayton) [inline src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1]: Please remove this cmdlet from the two psd1 files

File: tools/StaticAnalysis/Exceptions/AzureRM.Storage/SignatureIssues.csv [hunk 2]
```diff
@@ -9,3 +9,4 @@
...
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Storage\Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.RemoveAzureStorageAccountManagementPolicyCommand","Remove-AzureRmStorageAccountManagementPolicy","1","8600","Cmdlet 'Remove-AzureRmStorageAccountManagementPolicy' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Storage\Microsoft.Azure.Commands.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.GetAzureStorageAccountCommand","Get-AzureRmStorageAccount","1","8410","Parameter IncludeGeoReplicationStats of cmdlet Get-AzureRmStorageAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```
Comment (maddieclayton) [inline tools/StaticAnalysis/Exceptions/AzureRM.Storage/SignatureIssues.csv]: Let's quickly discuss in the meeting whether this parameter should be plural - I did not notice in the cmdlet review, which was my fault.

File: src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1 (hunk 1 above)
Comment (maddieclayton) [reply src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1]: Please also add a changelog update.

Comment (blueww) [reply src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1]: Removed from the 2 psd1 file. And change log updated. (change log already added for release for failover, but not released, so just update it)

File: tools/StaticAnalysis/Exceptions/AzureRM.Storage/SignatureIssues.csv (hunk 2 above)
Comment (blueww) [reply tools/StaticAnalysis/Exceptions/AzureRM.Storage/SignatureIssues.csv]: IncludeGeoReplicationStats, means Include GeoReplication Status. This is just the name aligned with SDK and rest api, suggest not change it.

Comment (blueww) [reply tools/StaticAnalysis/Exceptions/AzureRM.Storage/SignatureIssues.csv]: As we discussed in the meeting ,this should be good for switch parameter, and don't need change it.

**Comment by maddieclayton** [comment]
@blueww Will merge this when passing, we can release AzureRM.Storage on Monday, and hopefully get the updated package that is required for releasing Az.Storage later in the week.

**Pull request merged**